media/css/mozorg/contribute/contribute-2020.scss

```diff
@@ -0,0 +1,387 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../protocol/css/includes/lib';
+@import '../../../protocol/css/components/newsletter-form';
+@import '../../../protocol/css/templates/card-layout';
+
+.contribute-intro {
+    overflow: hidden;
+    padding-bottom: 90%;
+    padding-top: 0;
+    position: relative;
+
+    @media #{$mq-md} {
+        padding-bottom: 80%;
+    }
+
+    @media #{$mq-lg} {
+        padding-bottom: $layout-xl;
+    }
+
+    @media #{$mq-xl} {
+        padding-bottom: $layout-xl + $layout-md;
+    }
+
+    .contribute-intro-content {
+        @media #{$mq-lg} {
+            margin-top: $layout-xl;
+            max-width: 40%;
+        }
+    }
+
+    .contribute-intro-heading {
+        @include text-display-lg;
+        padding-bottom: $spacing-md;
+    }
+
+    .contribute-intro-media-wrapper {
+        @include bidi(((left, -10px, auto),(right, auto, -10px),));
+        position: absolute;
+        width: 100%;
+
+
+        @media #{$mq-lg} {
+            @include bidi(((right, -25px, auto),(left, auto, -25px),));
+            left: auto;
+            top: 0;
+            width: 50%;
+        }
+    }
+}
+.contribute-section-subheading {
+    @include text-title-xs;
+}
+
+.contribute-hero {
+    background-color: $color-marketing-gray-20;
+    margin-bottom: $layout-md;
+    padding-bottom: $layout-sm;
+    padding-top: $layout-sm;
+
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
+}
+
+.contribute-card-container {
+    @media #{$mq-sm} {
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    .contribute-card {
+        display: flex;
+        flex: 1 0 45%;
+
+        @media #{$mq-md} {
+            flex: auto;
+        }
+    }
+
+    .contribute-card-wrapper {
+        background: $color-white;
+        display: flex;
+        flex-direction: column;
+        transition: transform 150ms ease-in-out,
+            -webkit-transform 150ms ease-in-out;
```

Comment:
Do we need the `-webkit-transform` here still? If so, should it be using a separate `-webkit-transition`?